### ISSUE TITLE
Wrong (future) timestamps in tifffile-2022.4.8.tar.gz

### ISSUE THREAD

**Issue by mgorny** [cgohlke/tifffile#126]
When unpacking `tifffile-2022.4.8.tar.gz`, I get a lot of warnings of the kind:

```
tar: tifffile-2022.4.8/.gitattributes: time stamp 2022-04-09 00:57:46 is 7384.387871658 s in the future
tar: tifffile-2022.4.8/.gitignore: time stamp 2022-04-09 00:57:46 is 7384.387814399 s in the future
tar: tifffile-2022.4.8/ACKNOWLEDGEMENTS.rst: time stamp 2022-04-09 00:57:46 is 7384.387779652 s in the future
[...]
```

Given that `tar` stores timestamps as UTC, it seems that there is something wrong with the clock on the machine used to build sdists.

**Comment by cgohlke** [comment]
Looks like there is something off with ISO 8601 date parsing in the git command. Use the authoritative srcdist is on PyPI at https://pypi.org/project/tifffile/#files.

**Comment by cgohlke** [comment]
Fixed in [v2022.4.22](https://github.com/cgohlke/tifffile/commit/c97cd9c35494ce7ab941feac9fde055c47740f39)
